Detectors/TPC/calibration/src/IDCCCDBHelper.cxx

```diff
@@ -99,6 +104,284 @@ unsigned int o2::tpc::IDCCCDBHelper<DataT>::getUngroupedIndexGlobal(const unsign
+  hCside1D = IDCDrawHelper::drawSide(drawFun, fmt::format("{}", type.data()).data(), Side::C, nbins1D, xMin1D, xMax1D);
+
+  canv->Divide(2, 2);
+  canv->cd(1);
+  hAside2D->Draw("colz");
+  hAside2D->SetStats(0);
+  hAside2D->SetTitleOffset(1.05, "XY");
+  hAside2D->SetTitleSize(0.05, "XY");
+  o2::tpc::painter::drawSectorsXY(Side::A);
+
+  canv->cd(2);
+  hCside2D->Draw("colz");
+  hCside2D->SetStats(0);
+  hCside2D->SetTitleOffset(1.05, "XY");
+  hCside2D->SetTitleSize(0.05, "XY");
+  o2::tpc::painter::drawSectorsXY(Side::C);
+
+  canv->cd(3);
+  hAside1D->Draw();
+
+  canv->cd(4);
+  hCside1D->Draw();
+
+  return canv;
+}
+
+template <typename DataT>
+TCanvas* o2::tpc::IDCCCDBHelper<DataT>::drawIDCZeroRadialProfileHelper(TCanvas* outputCanvas, int nbinsY, float yMin, float yMax) const
+{
+  std::function<float(const unsigned int, const unsigned int, const unsigned int, const unsigned int)> idcFunc = [this](const unsigned int sector, const unsigned int region, const unsigned int irow, const unsigned int pad) {
+    return this->getIDCZeroVal(sector, region, irow, pad);
+  };
+
+  TCanvas* canv = nullptr;
+
+  if (outputCanvas) {
+    canv = outputCanvas;
+  } else {
+    canv = new TCanvas("c_sides_IDC0_radialProfile", "IDC0", 1000, 1000);
+  }
+
+  IDCDrawHelper::IDCDraw drawFun;
+  drawFun.mIDCFunc = idcFunc;
+  const std::string zAxisTitle = IDCDrawHelper::getZAxisTitle(IDCType::IDCZero);
+  TH2F* hAside2D = nullptr;
+  TH2F* hCside2D = nullptr;
+
+  double xBins[156];
+  xBins[0] = 848.5;
+
+  const int nRowsIROC = 63;  // pad size = 7.5 mm
+  const int nRowsOROC1 = 34; // pad size = 10 mm
+  const int nRowsOROC2 = 30; // pad size = 12 mm
+  const int nRowsOROC3 = 25; // pad size = 15 mm
+  const float gap1 = 25.75;  // gap between IROC and OROC1
+  const int gap2 = 21;       // gap between OROC1 and OROC2 and gap between OROC2 and OROC3
+
+  for (int i = 1; i < nRowsIROC + 1; i++) {
+    xBins[i] = xBins[0] + i * 7.5;
+  }
+  xBins[nRowsIROC + 1] = xBins[nRowsIROC] + gap1;
+
+  for (int i = nRowsIROC + 2; i < (nRowsIROC + 2 + nRowsOROC1); i++) {
+    xBins[i] = xBins[nRowsIROC + 1] + (i - (nRowsIROC + 2) + 1) * 10;
+  }
+  xBins[nRowsIROC + 2 + nRowsOROC1] = xBins[nRowsIROC + 1 + nRowsOROC1] + gap2;
+
+  for (int i = nRowsIROC + 3 + nRowsOROC1; i < (nRowsIROC + 3 + nRowsOROC1 + nRowsOROC2); i++) {
+    xBins[i] = xBins[nRowsIROC + 2 + nRowsOROC1] + (i - (nRowsIROC + 3 + nRowsOROC1) + 1) * 12;
+  }
+  xBins[nRowsIROC + 3 + nRowsOROC1 + nRowsOROC2] = xBins[nRowsIROC + 2 + nRowsOROC1 + nRowsOROC2] + gap2;
+
+  for (int i = nRowsIROC + 4 + nRowsOROC1 + nRowsOROC2; i < (nRowsIROC + 4 + nRowsOROC1 + nRowsOROC2 + nRowsOROC3); i++) {
+    xBins[i] = xBins[nRowsIROC + 3 + nRowsOROC1 + nRowsOROC2] + (i - (nRowsIROC + 4 + nRowsOROC1 + nRowsOROC2) + 1) * 15;
+  }
+
+  hAside2D = new TH2F("h_IDC0_radialProfile_Aside", "IDC0: Radial profile (A-Side)", 155, xBins, nbinsY, yMin, yMax);
```

Comment:
I added a function for the binning to the Painter: https://github.com/AliceO2Group/AliceO2/pull/8772
Once it is merged you can use it:
```suggestion
  const auto radialBinning = painter::getRowBinningCM();
  
  auto hAside2D = new TH2F("h_IDC0_radialProfile_Aside", "IDC0: Radial profile (A-Side)", radialBinning.size()-1, radialBinning.data(), nbinsY, yMin, yMax);
```